i/instana-java-sdk/instana-java-sdk_ubi_9.3.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package       : instana-java-sdk
+# Version       : 1.2.0
+# Source repo   : https://github.com/instana/instana-java-sdk.git
+# Tested on     : UBI 9.3
+# Language      : Java, Shell
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Pratibh Goshi<pratibh.goshi@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+set -e
+
+PACKAGE_NAME=instana-java-sdk/instana-java-sdk
```

Comment:
Package name should be instana-java-sdk (same as repo name). You can cd into the extra directory after cloning the package.